docs/standards/nft-2.0/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -13,83 +13,98 @@ sidebar_position: 4
+![LSP8 Address TokenIds Representation](/img/standards/lsp8-tokenId-address.jpeg)
 
-- _off-chain metadata_ (a **JSON file URL** stored on IPFS, a public registry, or even permissioned or private servers) or
-- _on-chain metadata_ (within another **ERC725Y smart contract**).
+TokenIds represented as **contract address** will allow the creation of more  **complex NFTs**, such as **Nested NFTs**, where each tokenId could be a contract that has its own storage using **[ERC725Y](../lsp-background//erc725.md#erc725y---generic-data-keyvalue-store)**, and the tokenId can **evolve** by changing metadata depending on **game rules**.  
 
-It is possible to know where the metadata for a specific `tokenId` is located by simply querying the LSP8 contract, giving the `tokenId` as an argument.
+![LSP8 Game Nested NFTs TokenIds Representation](/img/standards/lsp8-game.jpeg)
+### Unlimited Metadata
 
-One benefit of using an ERC725Y metadata contract per singular `tokenId` or NFT is that users can edit the metadata related to this NFT. Changeability makes the metadata flexible and upgradable. It enables the representation of unique NFTs that can be altered over time.
+:::success Recommendation
 
-## Types of `tokenId`
+To mark the **asset authenticity**, it's advised to use a combination between **[LSP4-DigitalAssetMetadata](./LSP4-Digital-Asset-Metadata.md)** and **[LSP12-IssuedAssets](../universal-profile/lsp12-issued-assets.md)**.
 
-:::caution
+:::
+
+The current token standards don't enable attaching metadata to the contract in a generic and flexible way, they set the **name**, **symbol**, and **tokenURI**. This is limiting for a digital asset that may want to express the creators, the community behind it, and to have the ability to update the metadata of the token and the tokenIds over time depending on a certain logic (Evolving tokens).  
+
+To ensure a flexible and generic asset representation, the token contract should use the **[LSP4-DigitalAsset-Metadata](./LSP4-Digital-Asset-Metadata.md)**. In this way, any information could be attached to the token contract.
+
+:::note Notice
 
-It is recommended that the `tokenId` should not change over the lifecycle of the LSP8 contract.
+The Metadata defined by the **ERC725Y Data Keys** can be set for **each tokenId**, not just for the whole NFT collection. Check **[LSP8 ERC725Y Data Keys](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#erc725y-data-keys)** in the LSP8 Standard Document.
 
 :::
 
-An LSP8 NFT can be represented in multiple ways, depending on the type of `tokenId` used as:
+### Force Boolean
 
-- a **number** (`uint256`) that increments on each newly minted NFT,
-- a unique `bytes32` value (representing a _serial number_, for instance), or
-- an ERC725Y contract address.
+It's estimated that more than **Hundreds of Millions of Dollars** worth of tokens were **sent to uncontrolled addresses** because of lacks of transfer validation checks.
```

Comment:
Again not sure who estimated this

Reply:
Agree on this. Like in academic papers, we should not cite without references in general.

Reply:
You guys are right, I will replace it with something else :)
https://medium.com/@maltabba/hundreds-of-millions-of-dollars-locked-at-ethereum-0x0-address-and-smart-contracts-addresses-how-4144dbe3458a